client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,83 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { Button, TextControl, ToggleControl } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import apiFetch from '@wordpress/api-fetch';
+import { sprintf, _n } from '@wordpress/i18n';
+
+class SubscriptionEdit extends Component {
+	state = {
+		subscriberCountString: '',
+	};
+
+	componentDidMount() {
+		// Get the subscriber count so it is available right away if the user toggles the setting
+		this.get_subscriber_count();
+	}
+
+	render() {
+		const { attributes, className, isSelected, setAttributes } = this.props;
+		const { subscribePlaceholder, showSubscribersTotal } = attributes;
+
+		if ( isSelected ) {
+			return (
+				<div className={ className } role="form">
+					<ToggleControl
+						label={ __( 'Show total subscribers' ) }
+						checked={ showSubscribersTotal }
+						onChange={ () => {
+							setAttributes( { showSubscribersTotal: ! showSubscribersTotal } );
+						} }
+					/>
+					<TextControl
+						placeholder={ subscribePlaceholder }
+						required
+						disabled={ true }
```

Comment:
```suggestion
						disabled
```

why we have `required`? 🤔